package.json

```diff
@@ -16,9 +16,11 @@
     "bcryptjs": "^2.0.0",
     "co": "^4.1.0",
     "cookie": "^0.1.2",
+    "email-templates": "^2.0.0-beta",
```

Comment:
I'm a little worried about using a beta, but I think this is ok until there is a release as we aren't using this much. Maybe create and issue to switch us to a release when its ready.